tests/driver_sps30/main.c

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ * @file
+ * @brief       Test application for the Sensirion SPS30 device driver
+ * @author      Michel Rottleuthner <michel.rottleuthner@haw-hamburg.de>
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "xtimer.h"
+#include "sps30.h"
+#include "sps30_params.h"
+
+#define TEST_START_DELAY_US          (2 * US_PER_SEC)
+#define SENSOR_RESET_DELAY_US        (10 * US_PER_SEC)
+#define SENSOR_STARTUP_DELAY_US      (10 * US_PER_SEC)
+#define POLL_FOR_READY_US            (1000000U)
```

Comment:
looks like you could go for seconds and use `xtimer_sleep` instead of `xtimer_usleep`?

Reply:
also the latter should be `1 * US_PER_SEC` otherwise?